xrdp/xrdp_keyboard.ini

```diff
@@ -58,6 +58,7 @@ layouts_map=default_layouts_map
 
 [default_rdp_layouts]
 rdp_layout_us=0x00000409
+rdp_layout_dvorak=0x00010409
```

Comment:
`rdp_layout_dvorak` is not bad but IMO, `rdp_layout_us_dvorak` is a plausible alternative.

MS document says it is US Dvorak:
https://docs.microsoft.com/ja-jp/windows-hardware/manufacture/desktop/windows-language-pack-default-values